applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/app_ZDebug.conf

```diff
@@ -83,6 +83,7 @@ CONFIG_SPEED_OPTIMIZATIONS=y
 
 CONFIG_USB=y
 CONFIG_USB_NRFX=y
+CONFIG_USB_NRFX_EVT_QUEUE_SIZE=8
```

Comment:
Hi @emob-nordic , I don't think setting this to 8 should give us any trouble. I cannot replicate anything bad with it.
But maybe you have some input here?

Reply:
@pdunaj It's really hard to say. Generally I would not decrease this number for Debug configuration, in fact You have this Konfig set to 16 below in this file (line 97). For Release configuration 8 should be sufficient. From my observations  it's better to keep that config higher if `CONFIG_USB_DEVICE_SOF` is enabled but this is not the case here.

Reply:
Thanks. Cleaned up configurations. Left 16 on debug (config applied after previous reduction) and reduced count on release only by 2.
Release must be higher or equal so as we usually test on debug.